src/ConstructiveSolidGeometry/PointsAndVectors/Points.jl

```diff
@@ -109,6 +109,8 @@ end
 @inline _convert_point(pt::AbstractCoordinatePoint, ::Type{Cylindrical}) = CylindricalPoint(pt)
 @inline _convert_point(pt::AbstractCoordinatePoint, ::Type{Cartesian}) = CartesianPoint(pt)
 
+scale(cart::CartesianPoint{T}, fact) where T = CartesianPoint{T}(cart.x*fact, cart.y*fact, cart.z*fact)
```

Comment:
I would be careful with this!
If we look at the docstring, `CartesianPoint` will **always** be saved in units of `m`.
I would prefer to have a constructor with units for `CartesianPoint` and solve this via the constructor and not with `scale`..